sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -61,6 +63,10 @@ class __SYCL_EXPORT command_graph {
     return add_impl(cgf, dep);
   }
 
+  // Adding USM allocation node:
+  node add_malloc(void*& ptr, size_t count, sycl::usm::alloc kind,
```

Comment:
Think rather than `count` this should be `numBytes`, since this function isn't templated on a type. (we also seem to be treating `count` as the number of bytes when allocating with `aligned_alloc`)